src/File.cpp

```diff
@@ -74,24 +75,45 @@ void TextFile::get_line(std::string& string) {
     }
 }
 
+void TextFile::skip_line() {
+    std::istream::sentry sentry(*this, true);
+    std::streambuf* buffer = std::istream::rdbuf();
```

Comment:
This line is a bit confusing. I believe that you are calling the function `rdbuf` in a parent class, but this is not obvious.

Reply:
yes, this is calling the function on the parent std::istream. I could also write it as `this->rdbuf()` instead.

Reply:
That would be preferable as it is easier to know what is going on.